code/game/machinery/doors/simple.dm

```diff
@@ -18,12 +18,17 @@
 	take_damage(100*material.combustion_effect(get_turf(src),temperature, 0.3))
 
 /obj/machinery/door/unpowered/simple/Initialize(mapload, var/material_name, var/locked)
-	. = ..(mapload)
 	if(!material_name)
 		material_name = MAT_STEEL
 	material = SSmaterials.get_material_datum(material_name)
 	if(!material)
 		return INITIALIZE_HINT_QDEL
+
+	if(!matter)
+		LAZYSET(matter, material.type, ceil(w_class * INITIAL_MATTER_W_CLASS_MULT * SHEET_MATERIAL_AMOUNT))
+
+	. = ..(mapload)
```

Comment:
Must always call parent even if you're returning a different value

Reply:
That or manually set the initialize flag

Reply:
It is calling parent.

Reply:
https://github.com/NebulaSS13/Nebula/pull/193/files#diff-5ad140ad80ac034ecfb5d029a22d973bR25 returns early